src/hct_mis_api/apps/payment/api/views.py

```diff
@@ -96,8 +98,9 @@ def get_queryset(self) -> QuerySet:
             program_cycle__program__in=program_ids,
         )
 
-    @etag_decorator(PaymentPlanKeyConstructor)
-    @cache_response(timeout=config.REST_API_TTL, key_func=PaymentPlanKeyConstructor())
+    # TODO: e2e failed probably because of cache here
+    # @etag_decorator(PaymentPlanKeyConstructor)
+    # @cache_response(timeout=config.REST_API_TTL, key_func=PaymentPlanKeyConstructor())
     def list(self, request: Request, *args: Any, **kwargs: Any) -> Response:
```

Comment:
@pkujawa I added the comment here that e2e failed because of cache sometimes... do you have any idea how to fix it? 
or I will try run tests with cache one more time just check the results

Reply:
looks work well 😄